src/LookingGlassEventBridge.js

```diff
@@ -0,0 +1,92 @@
+// https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement#events
+export const DOM_EVENTS = [
+  // ClipboardEvent
+  'copy',
+  'cut',
+  'paste',
+  'invalid',
+  // AnimationEvent
+  'animationcancel',
+  'animationend',
+  'animationiteration',
+  'animationstart',
+  // InputEvent
+  'beforeinput',
+  'input',
+  'change',
```

Comment:
These obviously don't apply to `HTMLCanvasElement`, but annoyingly many DOM events are sparsely documented in different specs.